README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">GPT4All</h1>
 
-<p align="center">GPT4All runs large language models (LLMs) privately on everyday desktops & laptops. <br> <br> No API calls or GPUs required - you can just download the application and <a href="https://docs.gpt4all.io/gpt4all_desktop/quickstart.html#quickstart">get started</a>
+<p align="center">GPT4All runs large language models (LLMs) privately on everyday desktops & laptops. <br> <br> No API calls or GPUs required - you can just download the application and <a href="https://docs.gpt4all.io/gpt4all_desktop/quickstart.html#quickstart">get started</a> <br> <br> Read about what's new in <a href="https://www.nomic.ai/blog/tag/gpt4all">our blog</a>.
```

Comment:
Could we put this somewhere below the video? That's where the rest of the links are.

I know that's past all of the download buttons, but if that's a problem I'd rather make them horizontal again like they [used to be](https://github.com/nomic-ai/gpt4all/pull/2488/commits/809dcbb05047cfa531271e2d615bff87a918d90c). Or we could move the other links to also be within the first screenful, like how [llama.cpp](https://github.com/ggerganov/llama.cpp#llamacpp) does it.

Reply:
I'd like this at the top so people visit the blog with our narrated GPT4All updates.